### ISSUE TITLE
Validation and model saving issues

### ISSUE THREAD

**Issue by gabrieledangeli** [bsc-quantic/tn4ml#17]
There are a couple of issues with the `tn4ml mnist_classification.ipynb` notebook when using `PatchEmbedding`:

- The `model.accuracy` function failed due to a tracer error caused by `jax.jit` parallelization.  
- The `model.save` functionality did not work within this notebook.

**Comment by emapuljak** [comment]
I'm on this!

**Comment by emapuljak** [comment]
Closed.
